docs/getting-started.md

```diff
@@ -112,10 +112,5 @@ Onderstaande tabellen bevatten testgevallen voor specifieke situaties waarmee de
 Tip: Je kan ook de [BAG Viewer](https://bagviewer.kadaster.nl/lvbag/bag-viewer/index.html){:target="_blank"} gebruiken om meer testgevallen te zoeken. 
 De (acceptatie)testomgeving bevat een kopie van de productiedata van enkele jaren geleden. Het is daardoor mogelijk dat recent toegevoegde of gewijzigde gegevens niet of onjuist worden getoond. Ook kan een recent toegevoegd adres wel verschijnen als adreszoekresultaat, maar niet worden geleverd door de API. Heb je voor het testen van je clientapplicatie volledige en actuele data nodig, dan kun je hiervoor je testomgeving [aansluiten op productie](#sluit-aan-op-productie).
```

Comment:
Ook in deze alinea wil je dan wellicht het e.e.a. aanpassen.

Reply:
Is dit niet een prima toevoeging?

Reply:
Je verwijst hier naar de acceptatie testomgeving maar de link heb je op een andere plaats uit dit document verwijderd. Lijkt me dat het dan niet zo veel zin heeft om hier wel naar die acceptatie omgeving te verwijzen, tenzij je hier weer de link opneemt.

Reply:
Ik zou hier inderdaad de acceptatietestomgeving helemaal niet meer noemen